From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alex Henning <elcron@gmail.com>
Date: Wed, 24 Sep 2014 22:38:04 -0400
Subject: [PATCH 0363/6262] Modified simulation debs to exclude building the
 toolchain.

Note: it currently doesn't seem to build due to a previous change that
needs further investigation.

Change-Id: I1a7f90a66d972b85541939f42e1afcb76f2fb9f4
---
 simulation/debs/Makefile                               | 10 +++++-----
 .../frcsim-eclipse-plugins/debian/control              |  2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/simulation/debs/Makefile b/simulation/debs/Makefile
index 71502f46271fd49132c5e37e115bf6169843f7ca..e24d8146e8fc5f4099e6e7724b2c5f8769444d91 100644
--- a/simulation/debs/Makefile
+++ b/simulation/debs/Makefile
@@ -27,17 +27,17 @@ allwpilib:
 	cd $(allwpilib) && mvn -T 8 clean package -Dwith-eclipse-plugins -DskipTests -DskipIT
 
 orig: orig-frcsim-gazebo-plugins orig-frcsim-gazebo-models orig-frcsim-eclipse-plugins \
-	  orig-frcsim-eclipse-toolchain-plugin orig-frcsim-libwpilibsim-cpp orig-frcsim
+	  orig-frcsim-libwpilibsim-cpp orig-frcsim
 
 debs: debs-frcsim-gazebo-plugins debs-frcsim-gazebo-models debs-frcsim-eclipse-plugins \
-	  debs-frcsim-eclipse-toolchain-plugin debs-frcsim-libwpilibsim-cpp debs-frcsim
+	  debs-frcsim-libwpilibsim-cpp debs-frcsim
 
 update-repository: debs clean-repository 
 	cd repository && reprepro includedeb $(codename) ../frcsim-gazebo-plugins/frcsim-gazebo-plugins_$(gazebo-plugins-package-version)_amd64.deb
 	cd repository && reprepro includedeb $(codename) ../frcsim-gazebo-plugins/frcsim-gazebo-plugins_$(gazebo-plugins-package-version)_i386.deb
 	cd repository && reprepro includedeb $(codename) ../frcsim-gazebo-models/frcsim-gazebo-models_$(gazebo-models-package-version)_all.deb
 	cd repository && reprepro includedeb $(codename) ../frcsim-eclipse-plugins/frcsim-eclipse-plugins_$(eclipse-plugins-package-version)_all.deb
-	cd repository && reprepro includedeb $(codename) ../frcsim-eclipse-toolchain-plugin/frcsim-eclipse-toolchain-plugin_$(eclipse-toolchain-package-version)_all.deb
+	# cd repository && reprepro includedeb $(codename) ../frcsim-eclipse-toolchain-plugin/frcsim-eclipse-toolchain-plugin_$(eclipse-toolchain-package-version)_all.deb
 	cd repository && reprepro includedeb $(codename) ../frcsim-libwpilibsim-cpp/frcsim-libwpilibsim-cpp_$(libwpilibsim-package-version)_amd64.deb
 	cd repository && reprepro includedeb $(codename) ../frcsim-libwpilibsim-cpp/frcsim-libwpilibsim-cpp_$(libwpilibsim-package-version)_i386.deb
 	cd repository && reprepro includedeb $(codename) ../frcsim/frcsim_$(frcsim-package-version)_all.deb
@@ -47,7 +47,7 @@ clean-repository:
 	cd repository && reprepro remove $(codename) frcsim-gazebo-plugins
 	cd repository && reprepro remove $(codename) frcsim-gazebo-models
 	cd repository && reprepro remove $(codename) frcsim-eclipse-plugins
-	cd repository && reprepro remove $(codename) frcsim-eclipse-toolchain-plugin
+	# cd repository && reprepro remove $(codename) frcsim-eclipse-toolchain-plugin
 	cd repository && reprepro remove $(codename) frcsim-libwpilibsim-cpp
 	cd repository && reprepro remove $(codename) frcsim
 
@@ -72,7 +72,7 @@ install:
 	sudo dpkg -i frcsim-gazebo-plugins/frcsim-gazebo-plugins_$(gazebo-plugins-package-version)_amd64.deb \
 	frcsim-gazebo-models/frcsim-gazebo-models_$(gazebo-models-package-version)_all.deb \
 	frcsim-eclipse-plugins/frcsim-eclipse-plugins_$(eclipse-plugins-package-version)_all.deb \
-	frcsim-eclipse-toolchain-plugin/frcsim-eclipse-toolchain-plugin_$(eclipse-toolchain-package-version)_all.deb \
+	# frcsim-eclipse-toolchain-plugin/frcsim-eclipse-toolchain-plugin_$(eclipse-toolchain-package-version)_all.deb \
 	frcsim-libwpilibsim-cpp/frcsim-libwpilibsim-cpp_$(libwpilibsim-package-version)_amd64.deb \
 	frcsim/frcsim_$(frcsim-package-version)_all.deb
 
diff --git a/simulation/debs/frcsim-eclipse-plugins/frcsim-eclipse-plugins/debian/control b/simulation/debs/frcsim-eclipse-plugins/frcsim-eclipse-plugins/debian/control
index 0b8eaee5c8690a70fe8a4c2ab1199626f48a0c0e..75b9f6036bbbe864d1dea13d53851674f55833fc 100644
--- a/simulation/debs/frcsim-eclipse-plugins/frcsim-eclipse-plugins/debian/control
+++ b/simulation/debs/frcsim-eclipse-plugins/frcsim-eclipse-plugins/debian/control
@@ -7,7 +7,7 @@ Build-Depends: debhelper (>= 9)
 
 Package: frcsim-eclipse-plugins
 Architecture: all
-Depends: ${shlibs:Depends}, ${misc:Depends}, eclipse, eclipse-jdt, eclipse-cdt, eclipse-rse, eclipse-cdt-launch-remote, frcsim-eclipse-toolchain-plugin, g++, libwebkitgtk-1.0-0, frcsim-libwpilibsim-cpp
+Depends: ${shlibs:Depends}, ${misc:Depends}, eclipse, eclipse-jdt, eclipse-cdt, eclipse-rse, eclipse-cdt-launch-remote, g++, libwebkitgtk-1.0-0, frcsim-libwpilibsim-cpp
 Description: FRC plugin for developing code for RoboRIO and WPILibSim.
  This plugin allows the usage of eclipse for development of WPILib
  robot code for the RoboRIO and simulation in gazebo.
\ No newline at end of file
